Fix expected diff check in integration tests

The integration tests that check expected output against a local fake server can fail based on the order of the output from the server. See runs: [fail](https://app.circleci.com/pipelines/github/GoogleCloudPlatform/opentelemetry-operations-go/1387/workflows/a7049ae8-6a76-4a78-9baa-a842d951eaee/jobs/1382), [pass](https://app.circleci.com/pipelines/github/GoogleCloudPlatform/opentelemetry-operations-go/1387/workflows/80e141fe-e8f8-4d46-982a-0bb6bd018737/jobs/1384) (from https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/418)

output:
```
   metrics_test.go:73: 
        	Error Trace:	metrics_test.go:73
        	Error:      	Expected requests fixture and actual GCM requests differ
        	Test:       	TestMetrics/Multi-project_metrics
        	Messages:   	  (*integrationtest.MetricExpectFixture)(Inverse(protocmp.Transform, protocmp.Message{
        	            	  	"@type":                             s"fixtures.MetricExpectFixture",
        	            	  	"create_metric_descriptor_requests": []protocmp.Message{{"@type": s"google.monitoring.v3.CreateMetricDescriptorRequest", "metric_descriptor": protocmp.Message{"@type": s"google.api.MetricDescriptor", "description": string("The approximate number of new series in this scrape"), "display_name": string("scrape_series_added"), "labels": []protocmp.Message{{"@type": s"google.api.LabelDescriptor", "key": string("service_instance_id")}, {"@type": s"google.api.LabelDescriptor", "key": string("service_name")}}, ...}, "name": string("projects/fakeprojectid")}, {"@type": s"google.monitoring.v3.CreateMetricDescriptorRequest", "metric_descriptor": protocmp.Message{"@type": s"google.api.MetricDescriptor", "display_name": string("ex_com_one"), "labels": []protocmp.Message{{"@type": s"google.api.LabelDescriptor", "key": string("A")}, {"@type": s"google.api.LabelDescriptor", "key": string("B")}, {"@type": s"google.api.LabelDescriptor", "key": string("C")}, {"@type": s"google.api.LabelDescriptor", "key": string("ex_com_lemons")}, ...}, "metric_kind": s"GAUGE", ...}, "name": string("projects/fakeprojectid")}, {"@type": s"google.monitoring.v3.CreateMetricDescriptorRequest", "metric_descriptor": protocmp.Message{"@type": s"google.api.MetricDescriptor", "display_name": string("ex_com_two"), "labels": []protocmp.Message{{"@type": s"google.api.LabelDescriptor", "key": string("A")}, {"@type": s"google.api.LabelDescriptor", "key": string("B")}, {"@type": s"google.api.LabelDescriptor", "key": string("C")}, {"@type": s"google.api.LabelDescriptor", "key": string("ex_com_lemons")}, ...}, "metric_kind": s"CUMULATIVE", ...}, "name": string("projects/fakeprojectid")}, {"@type": s"google.monitoring.v3.CreateMetricDescriptorRequest", "metric_descriptor": protocmp.Message{"@type": s"google.api.MetricDescriptor", "description": string("The scraping was successful"), "display_name": string("up"), "labels": []protocmp.Message{{"@type": s"google.api.LabelDescriptor", "key": string("service_instance_id")}, {"@type": s"google.api.LabelDescriptor", "key": string("service_name")}}, ...}, "name": string("projects/fakeprojectid")}, ...},
        	            	  	"create_time_series_requests": []protocmp.Message{
        	            	  		{
        	            	  			"@type":       s"google.monitoring.v3.CreateTimeSeriesRequest",
        	            	- 			"name":        string("projects/fake-other-project"),
        	            	+ 			"name":        string("projects/fakeprojectid"),
        	            	  			"time_series": []protocmp.Message{{"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"service_instance_id": "localhost:3333", "service_name": "demo2"}, "type": string("workload.googleapis.com/scrape_series_added")}, "metric_kind": s"GAUGE", "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "double_value": float64(22)}}}, ...}, {"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"ex_com_lemons": "13", "service_instance_id": "localhost:3333", "service_name": "demo2", "telemetry_sdk_language": "go", ...}, "type": string("workload.googleapis.com/ex_com_one")}, "metric_kind": s"GAUGE", "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "double_value": float64(1)}}}, ...}, {"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"A": "1", "B": "2", "C": "3", "ex_com_lemons": "10", ...}, "type": string("workload.googleapis.com/ex_com_one")}, "metric_kind": s"GAUGE", "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "double_value": float64(13)}}}, ...}, {"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"A": "1", "B": "2", "C": "3", "ex_com_lemons": "13", ...}, "type": string("workload.googleapis.com/ex_com_two")}, "metric_kind": s"CUMULATIVE", "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}, "start_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "distribution_value": protocmp.Message{"@type": s"google.api.Distribution", "bucket_counts": []int64{0, 0, 1, 0, ...}, "bucket_options": protocmp.Message{"@type": s"google.api.Distribution.BucketOptions", "explicit_buckets": protocmp.Message{"@type": s"google.api.Distribution.BucketOptions.Explicit", "bounds": []float64{1, 2, 5, 10, ...}}}, "count": int64(1), ...}}}}, ...}, ...},
        	            	  		},
        	            	  		{
        	            	  			"@type":       s"google.monitoring.v3.CreateTimeSeriesRequest",
        	            	- 			"name":        string("projects/fakeprojectid"),
        	            	+ 			"name":        string("projects/fake-other-project"),
        	            	  			"time_series": []protocmp.Message{{"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"service_instance_id": "localhost:3333", "service_name": "demo2"}, "type": string("workload.googleapis.com/scrape_series_added")}, "metric_kind": s"GAUGE", "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "double_value": float64(22)}}}, ...}, {"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"ex_com_lemons": "13", "service_instance_id": "localhost:3333", "service_name": "demo2", "telemetry_sdk_language": "go", ...}, "type": string("workload.googleapis.com/ex_com_one")}, "metric_kind": s"GAUGE", "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "double_value": float64(1)}}}, ...}, {"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"A": "1", "B": "2", "C": "3", "ex_com_lemons": "10", ...}, "type": string("workload.googleapis.com/ex_com_one")}, "metric_kind": s"GAUGE", "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "double_value": float64(13)}}}, ...}, {"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"A": "1", "B": "2", "C": "3", "ex_com_lemons": "13", ...}, "type": string("workload.googleapis.com/ex_com_two")}, "metric_kind": s"CUMULATIVE", "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}, "start_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "distribution_value": protocmp.Message{"@type": s"google.api.Distribution", "bucket_counts": []int64{0, 0, 1, 0, ...}, "bucket_options": protocmp.Message{"@type": s"google.api.Distribution.BucketOptions", "explicit_buckets": protocmp.Message{"@type": s"google.api.Distribution.BucketOptions.Explicit", "bounds": []float64{1, 2, 5, 10, ...}}}, "count": int64(1), ...}}}}, ...}, ...},
        	            	  		},
        	            	  	},
        	            	  	"self_observability_metrics": protocmp.Message{"@type": s"fixtures.SelfObservabilityMetric", "create_metric_descriptor_requests": []protocmp.Message{{"@type": s"google.monitoring.v3.CreateMetricDescriptorRequest", "metric_descriptor": protocmp.Message{"@type": s"google.api.MetricDescriptor", "description": string("Number of log records failed to be added to the sending queue."), "display_name": string("OpenCensus/exporter/enqueue_failed_log_records"), "labels": []protocmp.Message{{"@type": s"google.api.LabelDescriptor", "key": string("exporter")}}, ...}, "name": string("projects/myproject")}, {"@type": s"google.monitoring.v3.CreateMetricDescriptorRequest", "metric_descriptor": protocmp.Message{"@type": s"google.api.MetricDescriptor", "description": string("Number of metric points failed to be added to the sending queue."), "display_name": string("OpenCensus/exporter/enqueue_failed_metric_points"), "labels": []protocmp.Message{{"@type": s"google.api.LabelDescriptor", "key": string("exporter")}}, ...}, "name": string("projects/myproject")}, {"@type": s"google.monitoring.v3.CreateMetricDescriptorRequest", "metric_descriptor": protocmp.Message{"@type": s"google.api.MetricDescriptor", "description": string("Number of spans failed to be added to the sending queue."), "display_name": string("OpenCensus/exporter/enqueue_failed_spans"), "labels": []protocmp.Message{{"@type": s"google.api.LabelDescriptor", "key": string("exporter")}}, ...}, "name": string("projects/myproject")}, {"@type": s"google.monitoring.v3.CreateMetricDescriptorRequest", "metric_descriptor": protocmp.Message{"@type": s"google.api.MetricDescriptor", "description": string("Current size of the retry queue (in batches)"), "display_name": string("OpenCensus/exporter/queue_size"), "labels": []protocmp.Message{{"@type": s"google.api.LabelDescriptor", "key": string("exporter")}}, ...}, "name": string("projects/myproject")}, ...}, "create_time_series_requests": []protocmp.Message{{"@type": s"google.monitoring.v3.CreateTimeSeriesRequest", "name": string("projects/myproject"), "time_series": []protocmp.Message{{"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"status": "OK"}, "type": string("custom.googleapis.com/opencensus/googlecloudmonitoring/point_count")}, "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}, "start_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "int64_value": int64(18)}}}, "resource": protocmp.Message{"@type": s"google.api.MonitoredResource", "type": string("global")}}, {"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"grpc_client_method": "google.monitoring.v3.MetricService/CreateMetricDescriptor", "grpc_client_status": "OK"}, "type": string("custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs")}, "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}, "start_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "int64_value": int64(14)}}}, "resource": protocmp.Message{"@type": s"google.api.MonitoredResource", "type": string("global")}}, {"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"grpc_client_method": "google.monitoring.v3.MetricService/CreateTimeSeries", "grpc_client_status": "OK"}, "type": string("custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs")}, "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}, "start_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "int64_value": int64(2)}}}, "resource": protocmp.Message{"@type": s"google.api.MonitoredResource", "type": string("global")}}, {"@type": s"google.monitoring.v3.TimeSeries", "metric": protocmp.Message{"@type": s"google.api.Metric", "labels": map[string]string{"grpc_client_method": "google.monitoring.v3.MetricService/CreateMetricDescriptor"}, "type": string("custom.googleapis.com/opencensus/grpc.io/client/received_bytes_p"...)}, "points": []protocmp.Message{{"@type": s"google.monitoring.v3.Point", "interval": protocmp.Message{"@type": s"google.monitoring.v3.TimeInterval", "end_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}, "start_time": protocmp.Message{"@type": s"google.protobuf.Timestamp"}}, "value": protocmp.Message{"@type": s"google.monitoring.v3.TypedValue", "distribution_value": protocmp.Message{"@type": s"google.api.Distribution", "bucket_counts": []int64{0, 0, 0, 0, ...}, "bucket_options": protocmp.Message{"@type": s"google.api.Distribution.BucketOptions", "explicit_buckets": protocmp.Message{"@type": s"google.api.Distribution.BucketOptions.Explicit", "bounds": []float64{0, 1024, 2048, 4096, ...}}}}}}}, "resource": protocmp.Message{"@type": s"google.api.MonitoredResource", "type": string("global")}}, ...}}}},
        	            	  }))
--- FAIL: TestMetrics (10.80s)
```

the diff check should either not care about the ordering, or sort the output prior to comparing

--- a/exporter/collector/integrationtest/metrics_test.go
+++ b/exporter/collector/integrationtest/metrics_test.go
@@ -66,7 +66,10 @@ func TestMetrics(t *testing.T) {
 				return expectFixture.CreateTimeSeriesRequests[i].Name < expectFixture.CreateTimeSeriesRequests[j].Name
 			})
 			sort.Slice(expectFixture.CreateMetricDescriptorRequests, func(i, j int) bool {
-				return expectFixture.CreateMetricDescriptorRequests[i].Name < expectFixture.CreateMetricDescriptorRequests[j].Name
+				if expectFixture.CreateMetricDescriptorRequests[i].Name != expectFixture.CreateMetricDescriptorRequests[j].Name {
+					return expectFixture.CreateMetricDescriptorRequests[i].Name < expectFixture.CreateMetricDescriptorRequests[j].Name
+				}
+				return expectFixture.CreateMetricDescriptorRequests[i].MetricDescriptor.Name < expectFixture.CreateMetricDescriptorRequests[j].MetricDescriptor.Name
 			})
 			sort.Slice(expectFixture.CreateServiceTimeSeriesRequests, func(i, j int) bool {
 				return expectFixture.CreateServiceTimeSeriesRequests[i].Name < expectFixture.CreateServiceTimeSeriesRequests[j].Name
@@ -84,7 +87,10 @@ func TestMetrics(t *testing.T) {
 				return fixture.CreateTimeSeriesRequests[i].Name < fixture.CreateTimeSeriesRequests[j].Name
 			})
 			sort.Slice(fixture.CreateMetricDescriptorRequests, func(i, j int) bool {
-				return fixture.CreateMetricDescriptorRequests[i].Name < fixture.CreateMetricDescriptorRequests[j].Name
+				if fixture.CreateMetricDescriptorRequests[i].Name != fixture.CreateMetricDescriptorRequests[j].Name {
+					return fixture.CreateMetricDescriptorRequests[i].Name < fixture.CreateMetricDescriptorRequests[j].Name
+				}
+				return fixture.CreateMetricDescriptorRequests[i].MetricDescriptor.Name < fixture.CreateMetricDescriptorRequests[j].MetricDescriptor.Name
 			})
 			sort.Slice(fixture.CreateServiceTimeSeriesRequests, func(i, j int) bool {
 				return fixture.CreateServiceTimeSeriesRequests[i].Name < fixture.CreateServiceTimeSeriesRequests[j].Name

--- a/exporter/collector/integrationtest/testcases/testcase.go
+++ b/exporter/collector/integrationtest/testcases/testcase.go
@@ -366,7 +366,10 @@ func NormalizeMetricFixture(t testing.TB, fixture *protos.MetricExpectFixture) {
 		return fixture.CreateTimeSeriesRequests[i].Name < fixture.CreateTimeSeriesRequests[j].Name
 	})
 	sort.Slice(fixture.CreateMetricDescriptorRequests, func(i, j int) bool {
-		return fixture.CreateMetricDescriptorRequests[i].Name < fixture.CreateMetricDescriptorRequests[j].Name
+		if fixture.CreateMetricDescriptorRequests[i].Name != fixture.CreateMetricDescriptorRequests[j].Name {
+			return fixture.CreateMetricDescriptorRequests[i].Name < fixture.CreateMetricDescriptorRequests[j].Name
+		}
+		return fixture.CreateMetricDescriptorRequests[i].MetricDescriptor.Name < fixture.CreateMetricDescriptorRequests[j].MetricDescriptor.Name
 	})
 	sort.Slice(fixture.CreateServiceTimeSeriesRequests, func(i, j int) bool {
 		return fixture.CreateServiceTimeSeriesRequests[i].Name < fixture.CreateServiceTimeSeriesRequests[j].Name

--- a/exporter/collector/integrationtest/testdata/fixtures/metrics/basic_prometheus_metrics_expect.json
+++ b/exporter/collector/integrationtest/testdata/fixtures/metrics/basic_prometheus_metrics_expect.json
@@ -356,25 +356,6 @@
     {
       "name": "projects/fakeprojectid",
       "metricDescriptor": {
-        "name": "scrape_series_added",
-        "type": "workload.googleapis.com/scrape_series_added",
-        "labels": [
-          {
-            "key": "service_instance_id"
-          },
-          {
-            "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "description": "The approximate number of new series in this scrape",
-        "displayName": "scrape_series_added"
-      }
-    },
-    {
-      "name": "projects/fakeprojectid",
-      "metricDescriptor": {
         "name": "ex_com_one",
         "type": "workload.googleapis.com/ex_com_one",
         "labels": [
@@ -453,25 +434,6 @@
     {
       "name": "projects/fakeprojectid",
       "metricDescriptor": {
-        "name": "up",
-        "type": "workload.googleapis.com/up",
-        "labels": [
-          {
-            "key": "service_instance_id"
-          },
-          {
-            "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "description": "The scraping was successful",
-        "displayName": "up"
-      }
-    },
-    {
-      "name": "projects/fakeprojectid",
-      "metricDescriptor": {
         "name": "scrape_duration_seconds",
         "type": "workload.googleapis.com/scrape_duration_seconds",
         "labels": [
@@ -487,6 +449,25 @@
         "unit": "seconds",
         "description": "Duration of the scrape",
         "displayName": "scrape_duration_seconds"
+      }
+    },
+    {
+      "name": "projects/fakeprojectid",
+      "metricDescriptor": {
+        "name": "scrape_samples_post_metric_relabeling",
+        "type": "workload.googleapis.com/scrape_samples_post_metric_relabeling",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "description": "The number of samples remaining after metric relabeling was applied",
+        "displayName": "scrape_samples_post_metric_relabeling"
       }
     },
     {
@@ -511,8 +492,8 @@
     {
       "name": "projects/fakeprojectid",
       "metricDescriptor": {
-        "name": "scrape_samples_post_metric_relabeling",
-        "type": "workload.googleapis.com/scrape_samples_post_metric_relabeling",
+        "name": "scrape_series_added",
+        "type": "workload.googleapis.com/scrape_series_added",
         "labels": [
           {
             "key": "service_instance_id"
@@ -523,8 +504,27 @@
         ],
         "metricKind": "GAUGE",
         "valueType": "DOUBLE",
-        "description": "The number of samples remaining after metric relabeling was applied",
-        "displayName": "scrape_samples_post_metric_relabeling"
+        "description": "The approximate number of new series in this scrape",
+        "displayName": "scrape_series_added"
+      }
+    },
+    {
+      "name": "projects/fakeprojectid",
+      "metricDescriptor": {
+        "name": "up",
+        "type": "workload.googleapis.com/up",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "description": "The scraping was successful",
+        "displayName": "up"
       }
     }
   ],

--- a/exporter/collector/integrationtest/testdata/fixtures/metrics/metrics_multi_project_expected.json
+++ b/exporter/collector/integrationtest/testdata/fixtures/metrics/metrics_multi_project_expected.json
@@ -707,78 +707,40 @@
     {
       "name": "projects/fake-other-project",
       "metricDescriptor": {
-        "name": "scrape_series_added",
-        "type": "workload.googleapis.com/scrape_series_added",
+        "name": "ex_com_one",
+        "type": "workload.googleapis.com/ex_com_one",
         "labels": [
+          {
+            "key": "A"
+          },
+          {
+            "key": "B"
+          },
+          {
+            "key": "C"
+          },
+          {
+            "key": "ex_com_lemons"
+          },
           {
             "key": "service_instance_id"
           },
           {
             "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "description": "The approximate number of new series in this scrape",
-        "displayName": "scrape_series_added"
-      }
-    },
-    {
-      "name": "projects/fake-other-project",
-      "metricDescriptor": {
-        "name": "scrape_samples_scraped",
-        "type": "workload.googleapis.com/scrape_samples_scraped",
-        "labels": [
-          {
-            "key": "service_instance_id"
           },
           {
-            "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "description": "The number of samples the target exposed",
-        "displayName": "scrape_samples_scraped"
-      }
-    },
-    {
-      "name": "projects/fake-other-project",
-      "metricDescriptor": {
-        "name": "scrape_duration_seconds",
-        "type": "workload.googleapis.com/scrape_duration_seconds",
-        "labels": [
-          {
-            "key": "service_instance_id"
+            "key": "telemetry_sdk_language"
           },
           {
-            "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "unit": "seconds",
-        "description": "Duration of the scrape",
-        "displayName": "scrape_duration_seconds"
-      }
-    },
-    {
-      "name": "projects/fake-other-project",
-      "metricDescriptor": {
-        "name": "up",
-        "type": "workload.googleapis.com/up",
-        "labels": [
-          {
-            "key": "service_instance_id"
+            "key": "telemetry_sdk_name"
           },
           {
-            "key": "service_name"
+            "key": "telemetry_sdk_version"
           }
         ],
         "metricKind": "GAUGE",
         "valueType": "DOUBLE",
-        "description": "The scraping was successful",
-        "displayName": "up"
+        "displayName": "ex_com_one"
       }
     },
     {
@@ -823,6 +785,102 @@
     {
       "name": "projects/fake-other-project",
       "metricDescriptor": {
+        "name": "scrape_duration_seconds",
+        "type": "workload.googleapis.com/scrape_duration_seconds",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "unit": "seconds",
+        "description": "Duration of the scrape",
+        "displayName": "scrape_duration_seconds"
+      }
+    },
+    {
+      "name": "projects/fake-other-project",
+      "metricDescriptor": {
+        "name": "scrape_samples_post_metric_relabeling",
+        "type": "workload.googleapis.com/scrape_samples_post_metric_relabeling",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "description": "The number of samples remaining after metric relabeling was applied",
+        "displayName": "scrape_samples_post_metric_relabeling"
+      }
+    },
+    {
+      "name": "projects/fake-other-project",
+      "metricDescriptor": {
+        "name": "scrape_samples_scraped",
+        "type": "workload.googleapis.com/scrape_samples_scraped",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "description": "The number of samples the target exposed",
+        "displayName": "scrape_samples_scraped"
+      }
+    },
+    {
+      "name": "projects/fake-other-project",
+      "metricDescriptor": {
+        "name": "scrape_series_added",
+        "type": "workload.googleapis.com/scrape_series_added",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "description": "The approximate number of new series in this scrape",
+        "displayName": "scrape_series_added"
+      }
+    },
+    {
+      "name": "projects/fake-other-project",
+      "metricDescriptor": {
+        "name": "up",
+        "type": "workload.googleapis.com/up",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "description": "The scraping was successful",
+        "displayName": "up"
+      }
+    },
+    {
+      "name": "projects/fakeprojectid",
+      "metricDescriptor": {
         "name": "ex_com_one",
         "type": "workload.googleapis.com/ex_com_one",
         "labels": [
@@ -857,102 +915,6 @@
         "metricKind": "GAUGE",
         "valueType": "DOUBLE",
         "displayName": "ex_com_one"
-      }
-    },
-    {
-      "name": "projects/fake-other-project",
-      "metricDescriptor": {
-        "name": "scrape_samples_post_metric_relabeling",
-        "type": "workload.googleapis.com/scrape_samples_post_metric_relabeling",
-        "labels": [
-          {
-            "key": "service_instance_id"
-          },
-          {
-            "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "description": "The number of samples remaining after metric relabeling was applied",
-        "displayName": "scrape_samples_post_metric_relabeling"
-      }
-    },
-    {
-      "name": "projects/fakeprojectid",
-      "metricDescriptor": {
-        "name": "scrape_samples_post_metric_relabeling",
-        "type": "workload.googleapis.com/scrape_samples_post_metric_relabeling",
-        "labels": [
-          {
-            "key": "service_instance_id"
-          },
-          {
-            "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "description": "The number of samples remaining after metric relabeling was applied",
-        "displayName": "scrape_samples_post_metric_relabeling"
-      }
-    },
-    {
-      "name": "projects/fakeprojectid",
-      "metricDescriptor": {
-        "name": "scrape_samples_scraped",
-        "type": "workload.googleapis.com/scrape_samples_scraped",
-        "labels": [
-          {
-            "key": "service_instance_id"
-          },
-          {
-            "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "description": "The number of samples the target exposed",
-        "displayName": "scrape_samples_scraped"
-      }
-    },
-    {
-      "name": "projects/fakeprojectid",
-      "metricDescriptor": {
-        "name": "scrape_duration_seconds",
-        "type": "workload.googleapis.com/scrape_duration_seconds",
-        "labels": [
-          {
-            "key": "service_instance_id"
-          },
-          {
-            "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "unit": "seconds",
-        "description": "Duration of the scrape",
-        "displayName": "scrape_duration_seconds"
-      }
-    },
-    {
-      "name": "projects/fakeprojectid",
-      "metricDescriptor": {
-        "name": "up",
-        "type": "workload.googleapis.com/up",
-        "labels": [
-          {
-            "key": "service_instance_id"
-          },
-          {
-            "key": "service_name"
-          }
-        ],
-        "metricKind": "GAUGE",
-        "valueType": "DOUBLE",
-        "description": "The scraping was successful",
-        "displayName": "up"
       }
     },
     {
@@ -997,40 +959,59 @@
     {
       "name": "projects/fakeprojectid",
       "metricDescriptor": {
-        "name": "ex_com_one",
-        "type": "workload.googleapis.com/ex_com_one",
+        "name": "scrape_duration_seconds",
+        "type": "workload.googleapis.com/scrape_duration_seconds",
         "labels": [
-          {
-            "key": "A"
-          },
-          {
-            "key": "B"
-          },
-          {
-            "key": "C"
-          },
-          {
-            "key": "ex_com_lemons"
-          },
           {
             "key": "service_instance_id"
           },
           {
             "key": "service_name"
-          },
-          {
-            "key": "telemetry_sdk_language"
-          },
-          {
-            "key": "telemetry_sdk_name"
-          },
-          {
-            "key": "telemetry_sdk_version"
           }
         ],
         "metricKind": "GAUGE",
         "valueType": "DOUBLE",
-        "displayName": "ex_com_one"
+        "unit": "seconds",
+        "description": "Duration of the scrape",
+        "displayName": "scrape_duration_seconds"
+      }
+    },
+    {
+      "name": "projects/fakeprojectid",
+      "metricDescriptor": {
+        "name": "scrape_samples_post_metric_relabeling",
+        "type": "workload.googleapis.com/scrape_samples_post_metric_relabeling",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "description": "The number of samples remaining after metric relabeling was applied",
+        "displayName": "scrape_samples_post_metric_relabeling"
+      }
+    },
+    {
+      "name": "projects/fakeprojectid",
+      "metricDescriptor": {
+        "name": "scrape_samples_scraped",
+        "type": "workload.googleapis.com/scrape_samples_scraped",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "description": "The number of samples the target exposed",
+        "displayName": "scrape_samples_scraped"
       }
     },
     {
@@ -1050,6 +1031,25 @@
         "valueType": "DOUBLE",
         "description": "The approximate number of new series in this scrape",
         "displayName": "scrape_series_added"
+      }
+    },
+    {
+      "name": "projects/fakeprojectid",
+      "metricDescriptor": {
+        "name": "up",
+        "type": "workload.googleapis.com/up",
+        "labels": [
+          {
+            "key": "service_instance_id"
+          },
+          {
+            "key": "service_name"
+          }
+        ],
+        "metricKind": "GAUGE",
+        "valueType": "DOUBLE",
+        "description": "The scraping was successful",
+        "displayName": "up"
       }
     }
   ],

